docs/source/technical_tutorials/working_with_r/access_aws_maap.ipynb

```diff
@@ -0,0 +1,1588 @@
+{
```

Comment:
<p>since using <code>strsplit</code> with <code>tail</code> is rather cryptic, how about this instead:</p>

<pre class="ql-syntax" spellcheck="false">filename &lt;- sub(".*/", "", csv_key)  
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/471/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>